src/App/Kernel.php

```diff
@@ -482,18 +482,27 @@ public function validate(Request $request, Response $response)
                 $json->_validationInfo = $e->getMessage();
                 $json->_time = microtime(true) - $this->startTime;
                 $response->setContent(json_encode($json));
-
-                return $response;
             }
             $this->get('logger')->warning('Invalid JSON provided to user', [
                 'exception' => $e,
                 'request' => $request,
                 'response' => $response,
             ]);
         }
+
+        return $response;
     }
 
     public function cache(Request $request, Response $response)
     {
+        $response->setMaxAge($this->app['config']['ttl']);
```

Comment:
is there a way to override this for particular responses? (if needed in the future)